frameworks/Julia/Http.jl/run.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 
 for i in $(seq 0 $(($(nproc --all)-1))); do
-	julia server.jl
+	julia --threads auto server.jl &
```

Comment:
Does this make a difference in this benchmark, btw?

Reply:
The running multiple in processes in the background makes a big difference.  The threads might make some difference, but it's less pronounced if at all (no harm in having them though?).

Reply:
I would have expected that either would do (threads or multiprocess) if HTTP.jl is multithreaded.

As far as I can see in the code, tho, I think HTTP.jl doesn't start its own threads, but is expected to work if you call `listen` in multiple threads, so I wouldn't expect `--threads auto` to do much unless you change `server.jl` to use threads. OTOH, there's no point launching loads of threads if you're doing multiprocessing anyway (you probably want one thread or process  per core, not more).